server/server.js

```diff
@@ -1,14 +1,25 @@
 const express = require('express');
 const path = require('path');
-const MySQL = require('mysql');
+const mysql = require('mysql');
+const db = require('../db/connection.js');
 
+const statsQuery = `USE funding_stats; SELECT pledged, goal, currency, backers, 
+                    deadline, category, city, state, country FROM campaigns WHERE id = ?;`;
 const port = 3000;
 const app = express();
 
 app.use(express.static(path.resolve(__dirname, '../../funding-widget/public')));
 app.use(express.json());
 app.use(express.urlencoded({ extended: true }));
 
+app.get('/campaigns/:campaignId/stats', (req, res) => { // handle requests for a given campaign's stats
+  let campaign = parseInt(req.params.campaignId); // parse out the campaign's unique id
+
+  db.query(statsQuery, [campaign], (error, results) => { // retrieve campaign stats from the database
+    res.status(200).send(results[1][0]);
+  });
+});
+
 app.listen(port, () => {
   console.log(`Listening on port: ${port}`);
```

Comment:
nice factoring out the port :) which reminds me, I believe we all should be using different ports. Let's talk about it at next standup?

Reply:
Good idea.